docs/sources/tempo/metrics/_index.md

```diff
@@ -0,0 +1,104 @@
+---
+aliases:
+- /docs/tempo/latest/metrics-generator/prometheus-compatibility
+title: Prometheus compatibility
+menuTitle: Prometheus compatibility
+description: Learn about using tempo as a Prometheus data source.
+weight: 550
+keywords:
+  - Prometheus
+  - TraceQL
+---
+
+# Prometheus compatibility
+
+An experimental Prometheus-compatible API has been added to Tempo to provide a
+way to visualize metrics out of traces using TraceQL.
```

Comment:
If this is an experimental feature, should we add an [experimental warning](https://grafana.com/docs/writers-toolkit/write/shortcodes/#docsexperimental)? 

For example: 
```suggestion
{{< docs/experimental product="Prometheus-compatible API"  >}}

This experimental Prometheus-compatible API provides a way to visualize metrics out of traces using TraceQL.
```

The warning message looks like this : 
![image](https://github.com/grafana/tempo/assets/104772500/575ba847-d703-4b68-b35c-4a01197a0017)